includes/admin/views/html-order-shipment-summary.php

```diff
@@ -13,45 +14,41 @@
 $order_id    = $_POST["order_id"];
 $shipment_id = $_POST["shipment_id"];
 
-$order           = wc_get_order($order_id);
-$shipment        = WCMP()->export->get_shipment_data([$shipment_id], $order)[$shipment_id];
-$deliveryOptions = WCMP_Admin::getDeliveryOptionsFromOrder($order);
-$option_strings   = [
+$order = WCX::get_order($order_id);
+
+try {
+    $shipments       = WCMP()->export->get_shipment_data([$shipment_id], $order);
+    $deliveryOptions = WCMP_Admin::getDeliveryOptionsFromOrder($order);
+} catch (Exception $e) {
+    exit();
```

Comment:
Geen exit en al helemaal niet zonder iets erin. Graag een exception.
Waarom is een try catch eigenlijk nodig? Wat kan er mis gaan?

Reply:
Moest van editor omdat die functies mogelijk errors kunnen throwen, heb het weggehaald want er is zo gauw geen manier om het echt af te vangen